### PR TITLE
Refactor JS files to use consistent initialization pattern

### PR DESCRIPTION
**Pull request by nbeloglazov**
### Description

Most JS files today use initialization pattern where they check whether certain DOM element is present on the page and if so - initialize themselves. This PR updates all files to use the same pattern. This is required for upcoming JS bundling change where we'll compile all files together for admins and for applicants. So each JS file must handle the case where it's loaded on a page that doesn't actually "need" that file.

## Release notes

No-op refactoring of JS files for upcoming JS bundling chnage.

#### General

- [x] Added the correct label: < feature | bug | dependencies | infrastructure | ignore-for-release >
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

#### User visible changes

- [ ] Wrote browser tests using the [validateAccessibility](https://sourcegraph.com/github.com/civiform/civiform/-/blob/browser-test/src/support/index.ts?L437:14&subtree=true) method
- [ ] Manually tested at 200% size
- [ ] Manually evaluated tab order

#### New Features

- [ ] Add new FeatureFlag env vars to `server/conf/application.conf`
- [ ] Conditioned new functionality on a [FeatureFlag](https://docs.civiform.us/contributor-guide/developer-guide/feature-flags)
- [ ] Wrote browser tests with the feature flag off and on, etc.

### Issue(s) this completes

Issue #3864

### CODE REVIEW

File: server/app/assets/javascripts/azure_delete.ts [hunk 1]
```diff
@@ -3,8 +3,12 @@
  */
 class AzureDeleteController {
   private static FILEUPLOAD_DELETE_ID = 'fileupload-delete-button'
+  private static AZURE_UPLOAD = 'azure-upload'
```
Comment (bion) [inline server/app/assets/javascripts/azure_delete.ts:6]: suggestion (blocking): this value is being used as a query selector, but is missing a `.` at the beginning (assuming this is querying for an HTML class). Please add a test that fails when this selector is incorrect.

Comment (bion) [reply server/app/assets/javascripts/azure_delete.ts:6]: suggestion (blocking): please rename to `AZURE_UPLOAD_SELECTOR`

File: server/app/assets/javascripts/azure_upload.ts [hunk 2]
```diff
@@ -3,9 +3,13 @@
  */
 
 class AzureUploadController {
-  static FILEUPLOAD_FORM_ID = 'cf-block-form'
+  private static FILEUPLOAD_FORM_ID = 'cf-block-form'
+  private static AZURE_UPLOAD = '.azure-upload'
```
Comment (bion) [inline server/app/assets/javascripts/azure_upload.ts:7]: suggestion (blocking): please rename to `AZURE_UPLOAD_SELECTOR`

**Review by bion — CHANGES_REQUESTED** [review]

File: server/app/assets/javascripts/azure_upload.ts (hunk 2 above)
Comment (nbeloglazov) [reply server/app/assets/javascripts/azure_upload.ts]: Done.

File: server/app/assets/javascripts/azure_delete.ts (hunk 1 above)
Comment (nbeloglazov) [reply server/app/assets/javascripts/azure_delete.ts]: I can't add tests at the moment as we don't have infra for TS tests in server folder. I'll need to implement it first. Will do it in parallel.

**Review by bion — APPROVED** [review]

**Pull request merged**